itest/rust/src/rect2_test.rs

```diff
@@ -0,0 +1,102 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+use std::fmt::Debug;
+
+use crate::itest;
+use godot::prelude::{inner::InnerRect2, *};
```

Comment:
Don't use wildcard imports please (except in generated code where it would be very complex to name the types).